tools/training-benchmark-new-data.py

```diff
@@ -0,0 +1,169 @@
+import torch
```

Comment:
Please make this file similar to https://github.com/aiqm/torchani/blob/master/tools/training-benchmark.py, you don't have to use `ignite` as in that file, but it has to take the same arguments and print the same things, also benchmark on the same size networks.